tests/integration/test_logcollector/test_macos/test_macos_format_basic.py

```diff
@@ -5,13 +5,14 @@
 import os
 import pytest
 
-from time import sleep
 import wazuh_testing.logcollector as logcollector
 from wazuh_testing.tools.configuration import load_wazuh_configurations
 from wazuh_testing.remote import check_agent_received_message
-
+from wazuh_testing.tools.file import truncate_file
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.monitoring import FileMonitor
 # Marks
-pytestmark = pytest.mark.tier(level=1)
+pytestmark = [pytest.mark.darwin, pytest.mark.tier(level=1)]
```

Comment:
```suggestion
pytestmark = [pytest.mark.darwin, pytest.mark.tier(level=0)]
```
If this is a basic test, it should belong to the 0 tier.

Reply:
Done 58f4bb663ce1da50f8b24241a5ba741cebcffe72